scijava/scijava-ops-engine/src/main/java/org/scijava/ops/engine/yaml/AbstractYAMLOpInfoCreator.java

```diff
@@ -42,13 +65,108 @@ else if (p instanceof String) {
             }
         }
         // Create the OpInfo
+        OpInfo info;
         try {
-            return create(srcString, names, priority, null, version, yaml);
+            info = create(srcString, names, priority, null, version, tags);
         } catch (Exception e) {
             throw new RuntimeException(e);
         }
+        if (tags.containsKey("parameters")) {
+            List<Map<String, Object>> params =
+                (List<Map<String, Object>>) tags.get("parameters");
+            Iterator<Map<String, Object>> paramItr = params.iterator();
+
+            List<Member<?>> members = info.struct().members();
+            for (int i = 0; i < members.size(); i++) {
+                Member<?> m = members.get(i);
+                if (m.isInput() || m.isOutput()) {
+                    if (!paramItr.hasNext()) break;
+                    Map<String, Object> paramMap = paramItr.next();
+                    members.set(i, wrapMember.apply(m, paramMap));
+                }
+            }
+        }
+
+        return info;
```

Comment:
This feels like it could be improved - do we really have to wrap members? Can we not just create them correctly the first time?

Reply:
Ideally, we'd find a way to pass through parameter data in the `OpClassInfo` etc. constructor - but I think it's a change beyond the scope of this PR - I'll file a new PR.

Reply:
I wrote scijava/scijava#138 to address this.